src/content/docs/merchant/multistore.mdx

```diff
@@ -0,0 +1,185 @@
+
+folders:
+  /products/: /products/default
+  /en/products/: /en/products/default
+  /en_ca/products/: /en_ca/products/default`;
+export const fileName = 'fstab.yaml';
+export const highlights = ['file', 'CMS'];
+
+<Code code={exampleCode} lang="yaml" title={fileName} mark={highlights} />
+```
+
+## Store-specific files
+
+The content, configuration, and placeholder files help define the experience in each store view.
+
+### Content files
+
+The content files provide the structure of the pages that are served to your shoppers. are located in the store view directories and contain the content for each store view. These files are typically in the form of `.docx` files and are used to generate the store view pages.
+
+### Configuration files
+
+Each store view has its own set of configuration files. The `configs.xlsx` file defines the API endpoints, headers, and other environment-specific data for your production environment. The `configs-stage.xlsx` and `configs-dev.xlsx` files are used for staging and development environments, respectively.
+
+### Placeholders
+
+The `placeholders.xlsx` file stores reusable tokens for text and UI components. This file is used to define the text that is displayed on your storefront. The `placeholders.json` file is generated from this file and is used to replace the tokens in your content files. You can optionally create a `placeholders-overrides.json` file that define text that supersedes the values defined in the `en` store view.
+
+## Step-by-Step
+
+This example adds a new English Canadian store view (`en_ca`) that is similar to the default English store (`en`).
+
+<Tasks>
+
+<Task>
+### Add a new store
+
+Create an `en_ca` store view in the Commerce Admin and perform the following steps in your project.
+
+1. Create the `en_ca` folder under the project root in Sharepoint or Google Docs.
+
+1. Copy the configuration files, placeholders file, and document pages from the `en` directory to the `en_ca` directory.
+
+</Task>
+
+<Task>
+### Update Content Files
+
+Next, localize or translate the files in your project.
+
+1. Translate or localize content files in the `/en_ca/` folders.
+
+1. Ensure all labels in `placeholders.xlsx` are accurately translated into the desired language.
+
+</Task>
+
+<Task>
+### Update environment configurations
+
+Modify the three configuration files to include appropriate Commerce API endpoints and headers, such as the store view.
+
+</Task>
+
+<Task>
+### Set metadata to the chosen market's path
+
+Create a metadata file and configure settings for the new store view.
+
+1. Create a `metadata.xlsx` file in the root of your project.
+
+1. Set the **root** values to the directory where `configs.xlsx` files are located. The default is `/`.
+
+1. Set the **placeholders** to the `placeholders.json` file to use. The default is `/placeholders.json`.
+
+1. (Optional) Create a `placeholders_overrides.json` file if you have overrides that apply to the Canadian store.
+
```

Comment:
Additional (Optional) option:

1. (Optional) If you do not want to display a store switcher, publish an empty `store-switcher.docx` file.